otwcode/otwarchive
# AO3-5736 Update to error text when trying to log into a non-activated account.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5736

## Purpose

Currently if you set up an account but have not activated it yet, and you try to log in with that information, you receive a message in a red error bar that needed greater clarity to decreased user confusion.

## Testing Instructions

Activate a fresh account but don't confirm the email associated with it.
Attempt to log in.

## Credit

What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?
El

What pronouns do you prefer (she/her, he/him, zie/hir etc)?
They/Them
